tox.ini

```diff
@@ -3,18 +3,17 @@ envlist = py27, py34
 
 [testenv]
 deps =
-     pip==7.0.1
-     pytest==2.7.1
-     setuptools==16.0
-     wheel==0.24.0
-     flake8==2.4.1
-     py27: mock==1.3.0
+    pip==7.0.1
+    pytest==2.7.1
+    setuptools==16.0
+    wheel==0.24.0
+    flake8==2.4.1
+    numpy==1.9.2
+    scipy==0.16.0
+    pandas==0.16.2
+    scikit-learn==0.16.1
+    py27: mock==1.3.0
 
 commands =
-         pip install numpy --no-index
-         pip install scipy --no-index
-         pip install pandas --no-index
-         pip install scikit-learn --no-index
```

Comment:
There might have been a good reason to install these deps manually. I know I've run into issues building sklearn if script isn't already present.